crates/kitsune_p2p/dht_arc/src/dht_arc.rs

```diff
@@ -21,11 +21,30 @@ pub use dht_arc_bucket::*;
 #[cfg(any(test, feature = "test_utils"))]
 pub mod gaps;
 
-#[derive(Debug, Clone, Copy, serde::Serialize, serde::Deserialize, PartialEq, Eq, From, Into)]
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    serde::Serialize,
+    serde::Deserialize,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    Hash,
+    From,
+    Into,
+)]
 /// Type for representing a location that can wrap around
 /// a u32 dht arc
 pub struct DhtLocation(pub Wrapping<u32>);
 
+impl From<i32> for DhtLocation {
+    fn from(i: i32) -> Self {
+        (i as u32).into()
+    }
+}
```

Comment:
I'm not sure I'm comfortable with this... could we mark this `test_utils` feature only?